etc/build-examples.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+exit_code=0
+
+examples=("BugReport" "Docs" "KituraExample" "PerfectExample" "VaporExample")
+
+for example_project in ${examples[@]}; do
+    echo "Building $example_project"
+    example_dir="Examples/${example_project}"
+
+    # replace version string with master
+    etc/sed.sh -i 's/swift-driver", .upToNextMajor[^)]*)/swift-driver", .branch("master")/' "${example_dir}/Package.swift"
+
+    pushd "${example_dir}"
+
+    # don't exit on failure
+    set +e
+    swift build
+    build_success=$?
+    set -e
+
+    rm -rf ./.build
+    rm Package.resolved
+    git checkout Package.swift
+    popd
+
+    if [ ${build_success} -eq 0 ]; then
+        echo "================= Building $example_project succeeded ================="
+    else
+        echo "================= Building $example_project failed ================="
+        exit_code=1
```

Comment:
this was updated so that the script attempts to build all examples now, even if prior builds failed.